backend/src/main/java/atlantbh/restaurants/controllers/LocationController.java

```diff
@@ -0,0 +1,28 @@
+package atlantbh.restaurants.controllers;
+
+import atlantbh.restaurants.models.Location;
+import atlantbh.restaurants.services.LocationService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class LocationController extends BaseController<Location, LocationService> {
+
+    public ResponseEntity allCities(@PathVariable String country) {
+        return ResponseEntity.ok(service.findAllCitiesForCountry(country));
+    }
+    public ResponseEntity allCountries() {
+        return ResponseEntity.ok(service.findAllCountires());
+    }
+    public ResponseEntity topLocations(@RequestParam Integer size) {
+        try{
+            return ResponseEntity.ok(service.getTopLocations(size));
+        }
+        catch (Exception e){
```

Comment:
formatting.. it should be 
```
catch (Exception e) {
```

Reply:
Fixed 8c841a20e489a31c66d177c8682339f9d4f19de0